src/PerfView/StackViewer/CallTreeView.cs

```diff
@@ -301,6 +301,14 @@ private int IndexInParent(CallTreeNode node)
         #endregion
     }
 
+    public enum BackgroundColor
+    {
```

Comment:
Why do we need a new color enumeration?  Can't we simply use the string name for the color)?  
I think that might allow you to have only one DataTrigger items below.